pkg/ptrace/ptrace_linux_arm64.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2021 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//go:build cgo
+
+package ptrace
+
+import (
+	"encoding/binary"
+	"syscall"
+
+	"github.com/pkg/errors"
+)
+
+var endian = binary.LittleEndian
+
+const syscallInstrSize = 4
+
+const nrProcessVMReadv = 270
+const nrProcessVMWritev = 271
+
+func getIp(regs *syscall.PtraceRegs) uintptr {
+	return uintptr(regs.Pc)
+}
+
+// Syscall runs a syscall at main thread of process
+func (p *TracedProgram) Syscall(number uint64, args ...uint64) (uint64, error) {
```

Comment:
I suggest to add some comment about how to complete syscall with these code or why we should do these things. It is really difficult for me to check it.

Reply:
Add some comments. But for detailed information, you can read `man 2 syscall`. It's about the syscall convension.